dsl-interpreter/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "temporal-dsl-interpreter",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build",
+    "build.watch": "tsc --build --watch",
+    "start": "ts-node src/worker.ts",
+    "start.watch": "nodemon src/worker.ts",
+    "workflow1": "ts-node src/client.ts ../workflow1.yaml",
+    "workflow2": "ts-node src/client.ts ../workflow2.yaml",
+    "lint": "eslint ."
+  },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node"
+    },
+    "ext": "ts",
+    "watch": [
+      "src"
+    ]
+  },
+  "dependencies": {
+    "node-yaml": "^4.0.1",
+    "temporalio": "0.16.x"
+  },
+  "devDependencies": {
+    "@tsconfig/node14": "^1.0.0",
```

Comment:
We use node16 by default, where did you copy this from?

Reply:
default hello world